lib/rules/metadata/profile.js

```diff
@@ -68,8 +72,46 @@ exports.check = function(sr, done) {
                 });
             }
         }
-
         return done(meta);
+    }
+
+    var checkPreviousVersion = function (sr) {
+        return new PowerPromise(function (resolve) {
+            var $dl = sr.$("body div.head dl").first()
+            ,   dts = sr.extractHeaders($dl)
+            ,   $linkPrev = (dts.Previous) ? dts.Previous.dd.find("a").first() : ''
+            ,   shortname
+            ,   specExists
+            ;
+
+            var $linkLatest = (dts.Latest) ? dts.Latest.dd.find("a").first().attr('href') : '';
+            shortname = $linkLatest.match(/^https:\/\/www.w3.org\/TR\/(.+)\/$/)[1];
+            var req = sua.get('https://api.w3.org/specifications/' + shortname)
+                .set("User-Agent", "W3C-Pubrules/" + sr.version);
+            req.query({ apikey: process.env.W3C_API_KEY });
+            req.end(function (err, res) {
+                if (err || !res.ok) {
+                    if (res && res.status == 404) {
+                        specExists = false;
+                    }
+                } else {
+                    specExists = true;
+                }
+                resolve(specExists || $linkPrev);
+            });
+        });
+    };
+
+    if (id) {
+        if (/-NOTE/.test(id)) {
```

Comment:
```suggestion
        if (/\-NOTE/.test(id)) {
```

Reply:
After adding escape `\`, build failed: https://travis-ci.com/github/w3c/specberus/builds/166576468

```
/home/travis/build/w3c/specberus/lib/rules/metadata/profile.js
  105:14  error  Unnecessary escape character: \-  no-useless-escape
```

Reply:
Damn `eslint`! Ok let's skip that "useless" escape :)